src/frontends/onnx/README.md

```diff
@@ -0,0 +1,54 @@
+# ONNX FE
+
+The main responsibility of ONNX Frontend is importing ONNX models and conversion them to the `ov::Model` representation. 
+To capabilities of ONNX Frontend belong also:
+* modification of tensors properties (like data type and shapes)
+* changing topology of the models (like cutting subgraphs, inserting additional inputs and outputs)
+* user-friendly searching the models (via tensors and operators names)
+
+The component is written in `C++`, but `Python` bindings are also available.
+Each change to ONNX FE component requires `clang-format` code style and `NCC` naming style checks.
+
+
+## Key contacts
+
+In case of any questions, review and merge requests, please contact:
+* Tomasz Dołbniak (tomdol)
+* Mateusz Tabaka (mateusztabaka)
+* Mateusz Bencer (mbencer)
+
+from `openvino-onnx-frontend-maintainers` maintainers group.
+
+
+## Components
+
+ONNX Frontend implements an interface common for all frontends defined in [Frontends API](https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/common/include/openvino/frontend).
+The exported symbols are decorated with `ONNX_FRONTEND_API` flag. The key class responsible for importing is `FrontEnd` defined in (ONNX Frontend class)[https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/include/openvino/frontend/onnx/frontend.hpp].
```

Comment:
I think this sentence should be "prefixed" with an explanation that ONNX FE is a (shared) library.
```suggestion
The exported symbols are decorated with `ONNX_FRONTEND_API` macro. The key class responsible for importing is `FrontEnd` defined in (ONNX Frontend class)[https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/include/openvino/frontend/onnx/frontend.hpp].
```

Reply:
BTW I'm not sure I understand "The key class responsible for importing is `FrontEnd`" - can you rephrase it?

Reply:
I wanted to say like "the heart of the ONNX FE ecosystem if Frontend class", but now I'm thinking that it is rather useless info for user/contributor so I've removed this sentence.